docs/content/tutorial/step_07.ngdoc

```diff
@@ -1,380 +1,311 @@
+We will use Angular's {@link ng.$http $http} service in our controller for making an HTTP request to
+our web server to fetch the data in the `app/phones/phones.json` file. `$http` is just one of
+several built-in {@link guide/services Angular services} that handle common operations in web
+applications. Angular injects these services for you, right where you need them.
 
-Application routes in Angular are declared via the {@link ngRoute.$routeProvider $routeProvider},
-which is the provider of the {@link ngRoute.$route $route service}. This service makes it easy to
-wire together controllers, view templates, and the current URL location in the browser. Using this
-feature, we can implement [deep linking](http://en.wikipedia.org/wiki/Deep_linking), which lets us
-utilize the browser's history (back and forward navigation) and bookmarks.
+Services are managed by Angular's {@link guide/di DI subsystem}. Dependency injection helps to make
+your web applications both well-structured (e.g. separate entities for presentation, data, and
+control) and loosely coupled (dependencies between entities are not resolved by the entities
+themselves, but by the DI subsystem). As a result, applications are easier to test as well.
 
+<br />
+**`app/phone-list/phone-list.component.js:`**
 
-### A Note About DI, Injector and Providers
+```js
+  angular.
+    module('phoneList).
+    component('phoneList', {
+      templateUrl: 'phone-list/phone-list.template.html',
+      controller: function PhoneListController($http) {
+        var self = this;
+        self.orderProp = 'age';
+
+        $http.get('phones/phones.json').then(function(response) {
+          self.phones = response.data;
+        });
+      }
+    });
+```
 
-As you {@link tutorial/step_05 noticed}, {@link guide/di dependency injection} (DI) is at the core of
-AngularJS, so it's important for you to understand a thing or two about how it works.
+`$http` makes an HTTP GET request to our web server, asking for `phones.json` (the URL is relative
+to our `index.html` file). The server responds by providing the data in the JSON file.
+(The response might just as well have been dynamically generated by a backend server. To the
+browser and our app, they both look the same. For the sake of simplicity, we will use JSON files
+in this tutorial.)
 
-When the application bootstraps, Angular creates an injector that will be used to find and inject all
-of the services that are required by your app. The injector itself doesn't know anything about what
-`$http` or `$route` services do. In fact, the injector doesn't even know about the existence of these services
-unless it is configured with proper module definitions.
+The `$http` service returns a {@link ng.$q promise object}, which has a `then()` method. We call
+this method to handle the asynchronous response and assign the phone data to the controller, as a
+model called `phones`. Notice that Angular detected the JSON response and parsed it for us into the
```

Comment:
This is what they are called in the current version of the tutorial (I guess trying to make the connection with the MVC paradigm more obvious).
Changed it to `property`.